src/librustc_unicode/char.rs

```diff
@@ -551,6 +551,31 @@ impl char {
         }
     }
 
+    /// Returns true if this `char` is an a letter code point, and false if not.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// assert!('a'.is_letter());
+    /// assert!('京'.is_letter());
+    ///
+    /// let c = '💝';
+    /// // love is many things, but it is not a letter
+    /// assert!(!c.is_letter());
+    /// assert!(!'5'.is_letter());
+    /// ```
+    #[stable(feature = "rust1", since = "1.15.0")]
```

Comment:
1. First of all new additions are almost never instantly stable. You want a `unstable`.
2. You cannot use `rust1` feature name. Rather you need a name unique for this feature.